gpflow/mean_functions.py

```diff
@@ -102,19 +111,21 @@ def A(self, A):
     def b(self, b):
         pass
 
+
 class Constant(MeanFunction):
     """
     y_i = c,,
     """
     def __init__(self, c=None):
         MeanFunction.__init__(self)
         c = np.zeros(1) if c is None else c
+        c = np.reshape(c, (1, -1))
```

Comment:
Can you explain what is expected to happen if you pass in a vector-valued `c`?

Reply:
it will get reshaped to a vector having shape `[1, D]`. We need this first dimension for correct broadcasting.